apps/spruce/src/pages/spawn/spawnHost/EditSpawnHostModal.tsx

```diff
@@ -138,7 +165,7 @@ export const EditSpawnHostModal: React.FC<EditSpawnHostModalProps> = ({
       title="Edit Host Details"
       open={visible}
       data-cy="edit-spawn-host-modal"
-      submitDisabled={!hasChanges || loadingSpawnHost}
+      submitDisabled={!hasChanges || hasError || loadingSpawnHost}
```

Comment:
I spawned an expirable host and then tried editing it to a unexpirable host w/ a sleep schedule, but I couldn't get the save button to enable 🥹 I tried filling out all of the fields too

<img width="424" alt="Screenshot 2024-05-06 at 5 47 03 PM" src="https://github.com/evergreen-ci/ui/assets/47064971/c9e0d167-8ace-4e9c-bc56-1167ddf6f5e6">

Reply:
I think I still have this problem! This time I spawned an unexpirable host and when I went to edit the schedule, I couldn't save it